test/ng/directive/formSpec.js

```diff
@@ -460,6 +466,7 @@ describe('form', function() {
         .runs(function() {
           expect(doc.html()).toBe('');
           expect(destroyed).toBe(true);
+          expect(reloadPrevented).toBe('never called');
```

Comment:
The test results disagree with this expectation 
👀
⭕️ 

And based on the description I also would expect that this is true.